gmplot/gmplot.py

```diff
@@ -165,6 +166,42 @@ def _process_heatmap_kwargs(self, settings_dict):
 
         return settings_string
 
+    def groundoverlay(self, url, bounds_dict):
+
+        '''
+
+        :param url: Url of image to overlay
+        :param bounds_dict: dict of the form  {'north': , 'south': , 'west': , 'east': }
+        setting the image container
+        :return:
+
+        Example use:
+        import gmplot
+        gmap = gmplot.GoogleMapPlotter(37.766956, -122.438481, 13)
+        bounds_dict = {'north':37.832285, 'south': 37.637336, 'west': -122.520364, 'east': -122.346922}
+        gmap.groundoverlay('http://explore.museumca.org/creeks/images/TopoSFCreeks.jpg', bounds_dict)
+        gmap.draw("my_map.html")
+
+        Google Maps API documentation
+        https://developers.google.com/maps/documentation/javascript/groundoverlays#introduction
+
+        '''
+
```

Comment:
Can you remove some of the extra blank lines in this method and docstring? At least the double blank lines.